business_requirement/views/business_view.xml

```diff
@@ -9,6 +9,13 @@
             'search_default_parent_id': [active_id],
             'default_parent_id': active_id,
         }</field>
+        <field name="help" type="html">
```

Comment:
I think this is not correct. There is already a standard help description + this text is for project, not BR

Reply:
In short: keep the original helper message

Reply:
Yes, got it if project then it should be over here
https://github.com/OCA/business-requirement/blob/10.0/business_requirement/views/project.xml#L5
right?
@elicoidal

Reply:
No: I mean we should not touch the helper message for project and the one for BR is already fixed in the original module (so normally there is nothing to do in this module and you should remove this piece of code).
Let me know if unclear